featurebyte/routes/common/base_table.py

```diff
@@ -260,6 +261,8 @@ async def update_table_columns_info(
             Field to update
         data: Any
             Data to update
+        skip_semantic_check: bool
+            Flag to skip semantic check
```

Comment:
Would be good to add some info on why we need this flag and when it should be set.